src/smokeTest/java/uk/gov/hmcts/reform/draftstore/client/IdamClient.java

```diff
@@ -56,6 +58,19 @@ public void registerUser() {
             .statusCode(NO_CONTENT.value());
     }
 
+    /**
+     * Deletes the user with Idam's testing support.
+     */
+    public void deleteUser() {
+        RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(this.idamUrl)
+            .delete("/testing-support/accounts/{email}", email)
+            .then()
+            .statusCode(isOneOf(NO_CONTENT.value(), NOT_FOUND.value()));
+    }
```

Comment:
have you tested this with locally running idam? (given initial http method error pointed out by @doncem)

Reply:
if such resource exists (`DELETE /testing-support/accounts/{email}`) i'm happy to mark this as 🚀 . no_content stands for user existence and and not_found - not found hence no deletion happened. it's fine

Reply:
Yes, I wrote it based on what I've found in their code.